test/Icicle/Test/Source/Convert.hs

```diff
@@ -5,6 +5,7 @@
 {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
 module Icicle.Test.Source.Convert where
 
+import           Icicle.Test.Arbitrary
```

Comment:
yeah I don't know why it thinks just using an undefined splice is the same as "defining" it. This isn't Javascript....